.github/workflows/test-and-publish-nightly.yml

```diff
@@ -226,6 +245,29 @@ jobs:
           path: |
             build/test/logs/
             hs_err*.log
+      - name: Check core dumps on disk
+        id: check_core_dumps
+        if: ${{ failure() && !contains(matrix.runson.display, 'macos') }}
+        run: echo "core_dumps_count=$(ls core_* 2> /dev/null | wc -l)" >> $GITHUB_OUTPUT
+      - name: Print GDB stacktrace
+        if: ${{ failure() && steps.check_core_dumps.outputs.core_dumps_count != '0' && !contains(matrix.runson.display, 'macos') }}
+        shell: bash
+        run: |
+          apt update -y && apt install -y gdb
+          for core_file in $(ls core_*); do
+            echo "Printing stacktrace for $core_file"
+            # Remove 'core_' prefix
+            trimmed="${core_file#core_}"
+            # and PID suffix
+            trimmed="${trimmed%_[0-9]*}"
+            # Replace '!' with '/'
+            executable="${trimmed//\!//}"
+            asprof_address=$(gdb --batch -ex "info sharedlibrary" $executable $core_file | awk '/libasyncProfiler\.so/ && $1 ~ /^0x/ {print $1}')
+            gdb --batch \
+              -ex "add-symbol-file ${{ steps.extract_debug_artifact.outputs.debug_symbols }} $asprof_address" \
```

Comment:
Is the address arg required here?

Reply:
Yes, GDB complains when I don't provide it